CONTRIBUTING.md

```diff
@@ -37,8 +37,10 @@ $ git remote add fork https://github.com/YOUR_GITHUB_USERNAME/opencensus-python.
 Run tests:
 
 ```sh
-$ pip install nox  # Only first time.
-$ nox
+# Make sure you have all supported versions of Python installed
+# Currently these are 2.7, 3.4, 3.5, 3.6 and 3.7
```

Comment:
Probably no need to mention that all supported versions of Python again in the since we've already mentioned it in the "Prerequisites".